auth-web/src/components/auth/create-account/AccountCreateBasic.vue

```diff
@@ -14,14 +14,14 @@
         :label="govmAccount ? 'Ministry Name' : 'Account Name'"
         v-model.trim="orgName"
         :rules="orgNameRules"
-        :disabled="saving"
+        :disabled="saving || govmAccount"
```

Comment:
can we just replace govmAccount with readonly or something and check inside that method if its govm